pyscf/lib/misc.py

```diff
@@ -711,13 +711,13 @@ class call_in_background(object):
             afun1(a, b)
             do_something_else()
     '''
-    def __init__(self, *fns):
+
+    def __init__(self, *fns, sync=False):
```

Comment:
Python 2 does not support this statement.

We need either use **kwargs to include keyword "sync"  or change *fns to fns and update all callers.